### PR TITLE
[Server] Add server support to share WASM Filters

### PR DESCRIPTION
**Pull request by Chadha93**
**Notes for Reviewers**
Add server support to share WASM Filters

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7654b46a740d871870d3a3cfba40a68e023816c1 at: https://64c395d7c33bb066ad7b2824--meshery-docs-preview.netlify.app

File: server/handlers/user_handler.go [hunk 1]
```diff
@@ -193,3 +193,23 @@ func (h *Handler) ShareDesignHandler(w http.ResponseWriter, r *http.Request, _ *
...
+// Handle POST request for Sharing content
+//
+// Used to share filters with others
+// responses:
+// 	200:
+//  403:
+//  500:
+
+func (h *Handler) ShareFilterHandler(w http.ResponseWriter, r *http.Request, _ *models.Preference, _ *models.User, provider models.Provider) {
+	statusCode, err := provider.ShareDesign(r)
```
Comment (suhail34) [inline server/handlers/user_handler.go:207]:
```suggestion
	statusCode, err := provider.ShareFilter(r)
```
It will be shareFilter right?

**Review by suhail34 — CHANGES_REQUESTED** [review]

File: docs/_data/swagger.yml
Comment (leecalcote) [inline docs/_data/swagger.yml:1]: This file is autogenerated and shouldn't be included.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 5b09113eceb8ae01288e0f8b6424fcb4c3124798 at: https://64c838a952a8f403a1b2edd4--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
Merge conflicts

**Review by MUzairS15 — DISMISSED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit fdabeb82748a2e86f8fb02b17f4f4113e5ab52c6 at: https://64de2564ee581409761acb05--meshery-docs-preview.netlify.app

**Pull request merged**